src/test/java/com/atlassian/jira/cloud/jenkins/deploymentinfo/service/JiraDeploymentInfoSenderImplTest.java

```diff
@@ -240,15 +240,15 @@ public void testSendDeploymentInfo_whenNotAllowedEnvironmentType() {
 
     private JiraDeploymentInfoRequest createRequest() {
         return new JiraDeploymentInfoRequest(
-                SITE, ENVIRONMENT_ID, ENVIRONMENT_NAME, ENVIRONMENT_TYPE, mockWorkflowRun());
+                SITE, ENVIRONMENT_ID, ENVIRONMENT_NAME, ENVIRONMENT_TYPE, null, mockWorkflowRun());
```

Comment:
@jonathanpbailey would you be able to add an additional test covering the new changeSet parameter, i.e. to assert that the values from Jenkins ChangeSet and the passed changeSet parameter are properly merged?

Reply:
@rafalmyslek I intended on adding test coverage for this eventual, hence the reason I have not made a pull request.  Looks like @boussaffawalid made one before I could get to it.  I have no idea when I will get to this.

Reply:
Thanks for answering my questions. I'm happy to help with this PR, add the additional test coverage. Please let me know how if you are OK with that and how we should proceed with this change.